services/worker-manager/src/monitor.js

```diff
@@ -35,4 +35,25 @@ monitorManager.register({
   },
 });
 
+monitorManager.register({
+  name: 'workerError',
+  type: 'worker-error',
+  title: 'Worker Error',
+  level: 'warning',
```

Comment:
Maybe even just a notice? I'm happy either way.

Reply:
hm, makes sense -- if someone's looking for warnings, this might be a big spew of irrelevant stuff.